packages/components/psammead-timestamp/src/index.jsx

```diff
@@ -18,12 +18,13 @@ const PADDING = `
 `;
 
 const StyledTimestamp = styled.time`
-  ${props => (props.typographyFunc ? props.typographyFunc(props.script) : '')}
+  ${({ script, typographyFunc }) =>
+    script && (typographyFunc && typographyFunc(script))};
```

Comment:
```suggestion
    script && (typographyFunc && typographyFunc(script))}
```
Apparently this is causing linting to fail